keyboards/k34/k34.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 jp_smasher (@jp_smasher)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_K34( \
```

Comment:
Not a valid layout macro name
```suggestion
#define LAYOUT( \
```

Reply:
Done